addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3167,7 +3167,27 @@ msgctxt "#686"
 msgid "Verbose logging of [B]EPG[/B] component"
 msgstr ""
 
-#empty strings from id 687 to 699
+msgctxt "#687"
+msgid "From metadata"
+msgstr ""
+
+msgctxt "#688"
+msgid "90° clock-wise"
+msgstr ""
+
+msgctxt "#689"
+msgid "90° counter clock-wise"
+msgstr ""
+
+msgctxt "#690"
+msgid "180°"
```

Comment:
Can you re-use string 35230? Bonus points if you re-use 35228 - 35231. They don't include CW/CCW for conciseness and because trial-and-error isn't too painful with two possibilities. Otherwise, please comment strings.

Reply:
done. will not reuse 35228 as fernet specifically gave me the text to use.